apps/logging/signals.py

```diff
@@ -52,6 +52,18 @@ def handle_app_authorized(sender, request, auth_status, auth_status_code, user,
 
     # Get auth flow dict from session for logging
     auth_flow_dict = get_session_auth_flow_trace(request)
+    crosswalk_log = {}
+
+    try:
+        crosswalk_log = {
+            "id": user.crosswalk.id,
+            "user_hicn_hash": user.crosswalk.user_hicn_hash,
+            "user_mbi_hash": user.crosswalk.user_mbi_hash,
+            "fhir_id": user.crosswalk.fhir_id,
+            "user_id_type": user.crosswalk.user_id_type
+        }
+    except Exception:
```

Comment:
@nbragdon It is a best practice to handle an expected exception here and let the unexpected ones bubble up.

For example,

```python
import RelatedObjectDoesNotExist from django.contrib.auth.models
```
and 
```python
    except RelatedObjectDoesNotExist
```

Reply:
Per our discussion adding a TODO here for now